astro-client/types.go

```diff
@@ -18,7 +18,7 @@ type ResponseData struct {
 	GetClusters               []Cluster                    `json:"clusters,omitempty"`
 	SelfQuery                 *Self                        `json:"self,omitempty"`
 	RuntimeReleases           []RuntimeRelease             `json:"runtimeReleases,omitempty"`
-	DeploymentCreate          Deployment                   `json:"DeploymentCreate,omitempty"`
+	CreateDeployment          Deployment                   `json:"CreateDeployment,omitempty"`
```

Comment:
```suggestion
	CreateDeployment          Deployment                   `json:"createDeployment,omitempty"`
```
curious, is this working fine 🤔 or should it be `createDeployment` instead of `CreateDeployment` because it is defined as `createDeployment` https://github.com/astronomer/astro-cli/pull/695/files#diff-c6e601446e64ef83bcd4317f2fd57695772e1d5e22b79ea17f5b649ca40ba22eR40

Reply:
It seems to be working fine. I guess capitalization doesn't matter